ui/component/walletTipAmountSelector/view.jsx

```diff
@@ -304,16 +306,19 @@ function WalletTipAmountSelector(props: Props) {
 
       {/* lbc tab */}
       {activeTab === TAB_LBC && <WalletSpendableBalanceHelp />}
+
+      {/* help message */}
       {activeTab === TAB_FIAT &&
         (!hasCardSaved
           ? getHelpMessage(
               <>
                 <Button navigate={`/$/${PAGES.SETTINGS_STRIPE_CARD}`} label={__('Add a Card')} button="link" />
                 {' ' + __('To Tip Creators')}
-              </>
+              </>,
+              'add-a-card-help-message'
             )
           : !canReceiveFiatTip
-          ? getHelpMessage(__('Only creators that verify cash accounts can receive tips'))
+          ? getHelpMessage(__('Only creators that have added a bank account can receive tips'))
```

Comment:
I'm suggesting to change the copy here, it's unclear to me what a 'cash account' is and the tense is wrong as well

Reply:
Any time we do this, we have to be aware of translations, and having everyone retranslate. I'd only want to do this when absolutely necessary...I think it's fine as is.